### PR TITLE
Static route redistribution to IS-IS readme

### PR DESCRIPTION
**Pull request by rohit-rp**
https://github.com/openconfig/featureprofiles/issues/2494

- Static metric to ISIS Extended IP Reachability / IPv6 IP Reachability (TLV135 / TLV236)
- Redistribution to ISIS L2
- Static routes selected for redistribution base on combination of prefix-set and set-tag
- ISIS wide metric set to value of metric of static route (metric propagation)

### CODE REVIEW

**Review by rszarecki — DISMISSED** [review]
- We would need support for redistribute of  static route with "DROP" next-hop as well (in addition). Perhaps you can add it here easly. ike
```192.168.10.0/23``` next-hop ```"DROP"```

------------
LGTM

**Review by dplore — DISMISSED** [review]

**Comment by rohit-rp** [comment]
> * We would need support for redistribute of  static route with "DROP" next-hop as well (in addition). Perhaps you can add it here easly. ike
>   `192.168.10.0/23` next-hop `"DROP"`
> 
> LGTM

I dont think next-hop can be manipulated while redistributing to ISIS:

[/routing-policy/policy-definitions/policy-definition/statements/statement/actions/isis-actions/config](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-routing-policy.html#routing-policy-policy-definitions-policy-definition-statements-statement-actions-isis-actions-config)

It is applicable to BGP. I will add a subtest to static route redistribution to bgp

[/routing-policy/policy-definitions/policy-definition/statements/statement/actions/bgp-actions/config/set-next-hop](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-routing-policy.html#routing-policy-policy-definitions-policy-definition-statements-statement-actions-bgp-actions-config-set-next-hop)

File: feature/isis/static-route_isis_redistribution/README.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+    *   /network-instances/network-instance/table-connections/table-connection/state/address-family
+*   Verify source protocol is set to ```STATIC```
+    *   /network-instances/network-instance/table-connections/table-connection/state/src-protocol
+*   Verify destination protocol is set to ```ISIS```
+    *   /network-instances/network-instance/table-connections/table-connection/state/dst-protocol
+*   Verify default import policy is set to ```ACCEPT_ROUTE```
+    *   /network-instances/network-instance/table-connections/table-connection/state/default-import-policy
+*   Verify disable metric propogation is set to ```true```
+    *   /network-instances/network-instance/table-connections/table-connection/state/disable-metric-propagation
+*   Validate that the ATE receives the redistributed static route ```ipv4-route``` with default metric of ```0``` and not ```104```
```
[the remark this answers is not in the document]
Comment (rohit-rp) [reply feature/isis/static-route_isis_redistribution/README.md:87]: ok, MED can be either "missing" or "0" if not set explicitly depending on various scenarios (redistributed vs locally originated). It is "missing" for redistributed routes but I am not sure if all implementations follow the same standard, It is safe to validate either "missing" or "0" to pass the test

**Review by rszarecki — APPROVED** [review]

File: feature/isis/static-route_isis_redistribution/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dplore) [reply feature/isis/static-route_isis_redistribution/README.md:87]: Ok, this link is not needed.  I will open a PR for the OC yang model to clarify what the metric should be.

Ref:  https://github.com/openconfig/public/pull/1035

**Review by dplore — APPROVED** [review]

**Pull request merged**